pkg/uroot/uroot.go

```diff
@@ -327,49 +337,59 @@ func (o *Opts) addSymlinkTo(logger ulog.Logger, archive *initramfs.Opts, command
 	return nil
 }
 
-// resolvePackagePath finds import paths for a single import path or directory string
-func resolvePackagePath(logger ulog.Logger, env golang.Environ, pkg string) ([]string, error) {
-	// Search the current working directory, as well GOROOT and GOPATHs
-	prefixes := append([]string{""}, env.SrcDirs()...)
+// resolvePackagePath finds import paths for a single import path or directory string.
+//
+// In order of resolution:
+//
+//   ./foobar
```

Comment:
Well line 361 adds the prefix to the provided path with `filepath.Join()` which gives us something like "./foobar" and if a user provided us with a glob it will resolve that in line 362 with `filepath.Glob()` and allow things like "./foobar/glob" to work. Maybe a user just wants something from one particular path so no globbing used which in our case will then use just the path as provided. Or am I misunderstanding the question here?

Reply:
The comment confuses me, because it seems to imply that this code will try to do a glob, *in addition to the path that is passed in*, but it does not. I just find the comment very confusing.

Reply:
Ah now I got it. Not sure if "." was also an output of `filepath.Glob()` because in that case the comment would be true. Need to check that.

Reply:
I will reword the comment a bit to make things clearer.